packages/vitest/src/node/plugins/cssEnabler.ts

```diff
@@ -0,0 +1,31 @@
+import type { Plugin as VitePlugin } from 'vite'
+import { toArray } from '../../utils'
+import type { Vitest } from '../core'
+
+const isCss = (id: string) => {
```

Comment:
```suggestion
const isCSS = (id: string) => {
```

I personally prefer `css` and `CSS` whenever possible